plugins/woocommerce-admin/client/marketplace/components/extensions/extensions.tsx

```diff
@@ -17,60 +18,75 @@ import { Product, ProductType } from '../product-list/types';
 import { MARKETPLACE_ITEMS_PER_PAGE } from '../constants';
 
 interface ExtensionsProps {
+	categorySelector?: boolean;
 	products?: Product[];
 	perPage?: number;
+	label: string;
+	labelPlural: string;
 }
 
 export default function Extensions( props: ExtensionsProps ): JSX.Element {
 	const marketplaceContextValue = useContext( MarketplaceContext );
 	const { isLoading } = marketplaceContextValue;
+	const type =
+		props.label === 'extension' ? ProductType.extension : ProductType.theme;
```

Comment:
This looks like it could lead to issues.
Could we flip it? We can have `type` as a prop and define the labels inside the component.
This would also be consistent with some other components where the type is passed in to modify the component behavior.

Reply:
Yes, this makes way more sense! Will work on that for my next commit.

Reply:
Done in 6263b0025b807d0d3ea3089b0cb774bad59f61d2